pkg/server/server.go

```diff
@@ -117,3 +118,18 @@ func Start(errCh chan error) error {
 
 	return nil
 }
+
+func IsRunning() bool {
```

Comment:
I'm leaning more towards:
```go
func HealthCheck() error
```
The function would return an error that describes what's wrong with the server (or `nil` if it's healthy).

E.g. `API health check timed out`.